src/main/java/pro/sky/java/course2/weblibrary5/DepartmentController.java

```diff
@@ -0,0 +1,36 @@
+package pro.sky.java.course2.weblibrary5;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/departments")
+public class DepartmentController {
+    private final DepartmentServiceImpl departmentService;
+    public DepartmentController(DepartmentService departmentService) {
```

Comment:
Для лучшей читаемости поля отделяют пустой строкой от конструктора
```suggestion
    private final DepartmentServiceImpl departmentService;
    
    public DepartmentController(DepartmentService departmentService) {
```